src/app/sessions/components/session-edit-or-create-dialog/session-edit-or-create-dialog.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="sessions-create-container">
```

Comment:
What's the purpose of having such component? It looks like a carrier of data and dialog wrapper. I can see it sets the reloadDataOnInit flag to false. Is there anything else? Could we get rid of it? Besides, Why 'edit-or-create' ? You cannot edit a session with it, you can only create a new one